ci/ci/github.py

```diff
@@ -518,7 +548,7 @@ async def merge(self, gh):
             )
             return True
         except (gidgethub.HTTPException, aiohttp.client_exceptions.ClientResponseError):
-            log.info(f'merge {self.target_branch.branch.short_str()} {self.number} failed', exc_info=True)
+            log.exception(f'merge {self.target_branch.branch.short_str()} {self.number} failed')
```

Comment:
This isn't exceptional, AFAIK. A user sending us an unmergable branch is their fault, not ours.